src/bootloader-shared.ts

```diff
@@ -26,24 +26,26 @@
  */
 export const qrlResolver = (
   doc: Document,
-  eventUrl: string | null | undefined,
-  linkElm?: HTMLLinkElement,
-  href?: string,
-  url?: URL
+  element: Element | null,
+  eventUrl?: string | null,
+  _url?: string,
+  _base?: string | URL
 ): URL | undefined => {
-  if (eventUrl) {
-    url = new URL(
-      eventUrl.replace(/^(\w+):(\/)?/, (str, protocol, slash) => {
-        linkElm = doc.querySelector(`[rel="q.protocol.${protocol}"]`) as HTMLLinkElement;
-        href = linkElm && linkElm.href;
-        if (!href) error(protocol + ' not defined');
-        return href + (href!.endsWith('/') ? '' : slash || '');
-      }),
-      doc.baseURI
-    );
-    url.pathname += '.js';
+  if (eventUrl === undefined) {
```

Comment:
`if (eventUrl)` won't do, can we use `if (eventUrl == null)`?

Reply:
NO, it specifically is looking for `undefined`